test.yaml

```diff
@@ -0,0 +1,62 @@
+- schema_version: 1
```

Comment:
Is this test.yaml really required? If not, please remove it.

Other than that the PR LGTM.

Reply:
no, it is not, leftover from local tests.